lxl-web/src/lib/components/CookieConsent.svelte

```diff
@@ -35,15 +35,17 @@
 			}
 		},
 		onConsent: ({ cookie }) => {
-			if (cookie.categories.includes('analytics')) {
+			if ($matomoTracker && cookie.categories.includes('analytics')) {
 				$matomoTracker.rememberConsentGiven();
 			}
 		},
 		onChange: ({ cookie }) => {
-			if (cookie.categories.includes('analytics')) {
-				$matomoTracker.rememberConsentGiven();
-			} else {
-				$matomoTracker.forgetConsentGiven();
+			if ($matomoTracker) {
+				if (cookie.categories.includes('analytics')) {
+					$matomoTracker.rememberConsentGiven();
+				} else {
+					$matomoTracker.forgetConsentGiven();
```

Comment:
Not much of a difference but it feels like optional chaining could be used here instead of explicitly checking for `$matomoTracker` (e.g. `$matomoTracker?.rememberConsentGiven();`.

Reply:
Duh. Of course...